interface/modules/custom_modules/oe-module-weno/src/Services/TransmitProperties.php

```diff
@@ -188,9 +205,9 @@ public function cipherpayload()
     public function getProviderPassword()
     {
         $uid = $_SESSION['authUserID'];
-        $sql = "select setting_value from user_settings where setting_user = ? and setting_label = 'global:weno_provider_password'";
+        $sql = "select setting_value from user_settings where setting_user = ? and setting_label = 'global:weno_admin_password'";
         $prov_pass = sqlQuery($sql, [$uid]);
```

Comment:
should be able to drop above query since using th eglobal

Reply:
this looks like a refactor miss.